docs/en/siem/siem-ui.asciidoc

```diff
@@ -90,8 +106,8 @@ The map provides a visual overview of your network traffic. It is interactive,
 so you can start exploring data directly from the map. Hover over source and
 destination points to see more information, such as hostnames and IP addresses.
 To drill down, click a point and use the plus sign to add a field to the
-query bar. You can also click a hostname to jump to the SIEM Host page, or
-an IP address to open the relevant network details.
+query bar or drag a field to the Timeline. You can also click a hostname
```

Comment:
```suggestion
filter bar or drag a field to the Timeline. You can also click a hostname
```

Not sure if it's called a 'filter bar', but filters from the tooltips now add to the filter section below the query bar instead of directly as KQL.